libs/notifications/src/workflows/usage-limits/usage-limits.workflow.ts

```diff
@@ -0,0 +1,31 @@
+import { workflow } from '@novu/framework';
+import { z } from 'zod';
+import { renderUsageLimitsEmail } from './email';
+
+export const usageLimitsWorkflow = workflow(
+  'usage-limits',
+  async ({ step, payload }) => {
```

Comment:
Should we also send a simple in-app notification to the customer?

Reply:
Yes!